src/dali/abstract_ccxt_input_plugin.py

```diff
@@ -439,6 +439,16 @@ def _process_withdrawals(
             # End of pagination details
             pass
 
+    def _safe_api_call(
+        self,
+        function: Callable[..., Iterable[Dict[str, Union[str, float]]]],
+        params: Dict[str, Any],
+    ) -> Iterable[Dict[str, Union[str, float]]]:
+        return self.__safe_api_call(
+            function,
+            params
+        )
+
     def __safe_api_call(
```

Comment:
@eprbell should we just make this private method protected so that it is available for all classes?

Reply:
Yes, so that it's available to subclasses: makes sense.

Reply:
yeah, subclasses. That's what I meant. :)